spec/features/cases/offender_sar/case_editing_spec.rb

```diff
@@ -72,6 +76,37 @@
     then_i_should_see_the_pages_for_dispatch_reflected_on_the_show_page
   end
 
+  scenario 'user can edit reason for lateness' do
+    late_case = create_late_and_have_reason_for_lateness_case
+    cases_show_page.load(id: late_case.id)
+    
+    when_i_click_the_reason_for_lateness_change_link
+    then_change_reason_for_lateness_expect_reason_is_changed
+  end
+
+  def create_late_and_have_reason_for_lateness_case
+    late_case = create :offender_sar_case
+    late_case.reason_for_lateness = CategoryReference.list_by_category(:reasons_for_lateness).first
```

Comment:
@willmcb , yes, definitely a good idea, not sure whether you can remember, we have slightly discussed this idea a little bit,  the ideal model in my mind would be a more generic wrapper rather than defining a new one every time we have a new category being added into reference table.  for now,  let us go for this simple query way.   Maybe when we work on the data request table , we can refactor this part